### PR TITLE
Added Sign In feature for Web platform

### PR DESCRIPTION
**Pull request by AlexTsybulskiy**
Screenshot 2021-08-27 at 14 22 37

### CODE REVIEW

File: features/1. Sign In.feature [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+   When Input "<email>" text to "Email field" on "Sign In" page
+      And Input "<password>" text to "Password field" on "Sign In" page
+      And Click "Sign In button" on "Sign In" page
+   Then The "General Home" page is open
+      And The "<page name>" page is open
+   When Click "Log out button" on "General Home" page
+   Then The "Sign In" page is open
+
+   Examples: Users
+     | role         | email                      | password   | page name           |
```
Comment (IVeram22) [inline features/1. Sign In.feature:16]: Headers of Examples Tables must begin with Capital letters and don't contain special characters

File: features/1. Sign In.feature [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+   @sign_in @smoke @regression @Web
+  Scenario Outline: 1.5 Password error is displayed on Sign In page after login as <user> with invalid password
+   When Input "<email>" text to "Email field" on "Sign In" page
+      And Input "<invalid_password>" text to "Password field" on "Sign In" page
+      And Click "Sign In button" on "Sign In" page
+   Then The "Invalid username or password error" is displayed on "Sign In" page
+
+   Examples: Users
+     | user           | email                    | invalid_password  |
+     | invalid_user   | wnadm111@mailinator.com  |    blablabla      |
```
Comment (IVeram22) [inline features/1. Sign In.feature:84]: Tests 1.4 and 1.5 can be combined into 1

File: features/1. Sign In.feature [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      And Click "Sign In button" on "Sign In" page
+   Then The "Invalid username or password error" is displayed on "Sign In" page
+   When Clean input "<password>" text to "Password field" on "Sign In" page
+      And Click "Sign In button" on "Sign In" page
+   Then The "General Home" page is open
+      And The "<page name>" page is open
+
+   Examples: Users
+     | role       | email                    |  password   |  invalid_password  |     page name      |
+     | WN Admin   | wnadm111@mailinator.com  |  Qwerty123! |  blablabla         |   WN Admin's Home  |
```
Comment (IVeram22) [inline features/1. Sign In.feature:139]: Tests 1.8 and 1.9 can be combined into 1

File: features/1. Sign In.feature (hunk 2 above)
Comment (IVeram22) [inline features/1. Sign In.feature:84]: Entering the wrong password should be more appropriate, for example: P@ssword1

File: resources/LOCATORS.json [hunk 4]
```diff
@@ -1,8 +1,214 @@
 {
   "pages": [
+    {
+      "pageName": "Sign In",
+      "pageDescription": "Sign In page",
```
Comment (IVeram22) [inline resources/LOCATORS.json:5]: The description of the page should contain information about how we get to it. For example: We get to this page when we open the application

**Comment by AlexTsybulskiy** [comment]
Screenshot 2021-08-31 at 15 09 07

File: features/1. Sign In.feature [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+      And Click "Sign In button" on "Sign In" page
+   Then The "Invalid email error" is displayed on "Sign In" page
+   When Clean input "<Valid email>" text to "Email field" on "Sign In" page
+      And Clean input "<Invalid password>" text to "Password field" on "Sign In" page
+      And Click "Sign In button" on "Sign In" page
+   Then The "Invalid username or password error" is displayed on "Sign In" page
+
+   Examples: Users
+     | User      | Valid email             |  Invalid email    |  Valid password   | Invalid password  |
+     | WN Admin  | wnadm111@mailinator.com |  qsjd@wsjd@.com   |  Qwerty123!       |    P@ssword1      |
```
Comment (IVeram22) [inline features/1. Sign In.feature:76]: Some time you use "User" or "Role", can change it for "User role"

File: features/1. Sign In.feature [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+   Examples: Users
+     | User         | Email                      | Password   | Page name           |
+     | WN Admin     | wnadm111@mailinator.com    | Qwerty123! | WN Admin's Home     |
+
+
+   @sign_in @smoke @regression @Web
+  Scenario Outline: 1.9 Login with valid account as <role> after login with invalid email
+   When Input "<Invalid email>" text to "Email field" on "Sign In" page
+      And Input "<Valid password>" text to "Password field" on "Sign In" page
+      And Click "Sign In button" on "Sign In" page
```
Comment (IVeram22) [inline features/1. Sign In.feature:106]: 1.6, then 1.9?

**Comment by AlexTsybulskiy** [comment]
Screenshot 2021-08-31 at 16 31 21

**Pull request merged**